pkgs/by-name/cl/clevis/package.nix

```diff
@@ -67,24 +72,43 @@ stdenv.mkDerivation rec {
     tpm2-tools
   ];
 
-  outputs = [
-    "out"
-    "man"
-  ];
+  strictDeps = true;
+
+  postFixup = let
+    binPath = lib.makeBinPath [
+      coreutils
+      cryptsetup
+      gnugrep
+      gnused
+      jose
+      libpwquality
+      luksmeta
+      tpm2-tools
+    ];
+    # We wrap the main clevis binary entrypoint - but not the sub-command
+    # binaries
+  in ''
+    patchShebangs --host $out/bin/clevis
+    wrapProgram $out/bin/clevis \
+      --prefix PATH ':' "${binPath}:${placeholder "out"}/bin"
```

Comment:
Can't placeholder "out" be added to makeBinPath, too?

Reply:
> Can't placeholder "out" be added to makeBinPath, too?

From my experience it can't.